Lib/dis.py

```diff
@@ -413,6 +413,9 @@ def _get_instructions_bytes(code, varname_from_oparg=None,
             labels.add(target)
     starts_line = None
     for offset, op, arg in _unpack_opargs(code):
+        # get the next position before we skip a CACHE instruction
+        # to associate the correct position information. 
```

Comment:
Also, the CI is mad about this trailing whitespace. Didn't see it until now:

```suggestion
        # to associate the correct position information.
```